network/src/network/connection_manager.rs

```diff
@@ -326,9 +327,8 @@ impl ConnectionManager {
                 });
                 metrics::gauge!(METRIC_CONNECTIONS_PARTIAL).increment(1);
             }
-            Into0RttResult::InvalidConnection(e) => {
-                // TODO: Lower log level to trace/debug?
-                tracing::warn!(%remote_addr, "invalid incoming connection: {e}");
```

Comment:
've you ever seen this logs? Maybe it's ok?

Reply:
We have not yet spammed with invalid peer ids (malformed ed25519 pubkeys). It is the case when this error is visible, and we can just ignore that spam (no need to flood logs with it)